worlds/mm-recomp/Regions.py

```diff
@@ -10,14 +10,39 @@ class MMRRegionData(NamedTuple):
     "Menu": MMRRegionData(["Clock Town"]),
     "Clock Town": MMRRegionData(["Termina Field", "The Moon"]),
     "The Moon": MMRRegionData([]),
-    "Termina Field": MMRRegionData(["Southern Swamp", "Romani Ranch"]),
+    "Termina Field": MMRRegionData(["Southern Swamp", "Romani Ranch", "Path to Mountain Village", "Great Bay", "Road to Ikana"]),
     "Southern Swamp": MMRRegionData(["Southern Swamp (Deku Palace)"]),
     "Romani Ranch": MMRRegionData([]),
     "Southern Swamp (Deku Palace)": MMRRegionData(["Swamphouse", "Deku Palace", "Woodfall"]),
     "Swamphouse": MMRRegionData([]),
     "Deku Palace": MMRRegionData([]),
     "Woodfall": MMRRegionData(["Woodfall Temple"]),
     "Woodfall Temple": MMRRegionData([]),
+    "Path to Mountain Village": MMRRegionData(["Mountain Village"]),
+    "Mountain Village": MMRRegionData(["Twin Islands", "Path to Snowhead"]),
+    "Twin Islands": MMRRegionData(["Goron Village"]),
+    "Goron Village": MMRRegionData([]),
+    "Path to Snowhead": MMRRegionData(["Snowhead Temple"]),
+    "Snowhead Temple": MMRRegionData([]),
+    "Great Bay": MMRRegionData(["Ocean Spider House", "Pirates' Fortress", "Pinnacle Rock", "Zora Cape"]),
+    "Ocean Spider House": MMRRegionData([]),
+    "Pirates' Fortress (Exterior)": MMRRegionData(["Pirates' Fortress Sewers", "Pirates' Fortress"]),
+    "Pirates' Fortress Sewers": MMRRegionData(["Pirates' Fortress"]),
+    "Pirates' Fortress": MMRRegionData([]),
+    "Pinnacle Rock": MMRRegionData([]),
+    "Zora Cape": MMRRegionData(["Zora Hall", "Great Bay Temple"]),
+    "Zora Hall": MMRRegionData([]),
+    "Great Bay Temple": MMRRegionData([]),
+    "Road to Ikana": MMRRegionData(["Ikana Graveyard", "Ikana Canyon"]),
+    "Ikana Graveyard": MMRRegionData([]),
+    "Ikana Canyon": MMRRegionData(["Secret Shrine", "Beneath the Well", "Ikana Castle", "Stone Tower"]),
+    "Secret Shrine": MMRRegionData([]),
+    "Beneath the Well": MMRRegionData([]),
+    "Ikana Castle": MMRRegionData(["Beneath the Well]),
```

Comment:
there's a quote missing here

Reply:
Come to think of it, it would probably make more sense to go the other direction, wouldn't it? Give Beneath the Well an entrance into Ikana Castle, since that's the direction the player normally goes anyway.